cypress/e2e/brewery/FileParameters.cy.js

```diff
@@ -102,6 +108,26 @@ describe('Simple operations on a file parameter', () => {
     ScenarioParameters.save();
     BreweryParameters.getExampleDatasetPart1DownloadButton().should('not.exist');
   });
+
+  it('can download an uploaded file that has been renamed, after a page refresh', () => {
+    const breweryScenarioName = forgeScenarioName();
+    let breweryScenarioId;
+    scenarioNamesToDelete.push(breweryScenarioName);
+    Scenarios.createScenario(breweryScenarioName, true, SCENARIO_DATASET, BREWERY_RUN_TEMPLATE).then((value) => {
+      breweryScenarioId = value.scenarioCreatedId;
+      BreweryParameters.switchToFileUploadTab();
+      BreweryParameters.uploadInitialStock(FILE_PATH_1);
+      ScenarioParameters.save();
+      BreweryParameters.getInitialStockFileName().should('have.text', 'CSV file');
+      BreweryParameters.downloadInitialStock();
+      Downloads.checkByContent('initial_stock_dataset.csv', EXPECTED_DATA_AFTER_DUMMY_DATASET_1_UPLOAD);
+
+      route.browse({ url: `${BREWERY_WORKSPACE_ID}/scenario/${breweryScenarioId}` });
```

Comment:
Is the Is the goal of this action to refresh the page ? In this case, you can use `cy.reload()`

Reply:
Using `cy.reload()` would not use custom interceptions we have set in the `route.browse` function (e.g. interception of PowerBI token query)